package.json

```diff
@@ -21,22 +21,33 @@
   },
   "dependencies": {
     "@moxy/react-keyboard-only-outlines": "^1.0.1",
+    "@wordpress/a11y": "^2.7.0",
+    "@wordpress/api-fetch": "^3.11.0",
+    "@wordpress/components": "^9.2.2",
+    "@wordpress/dom-ready": "^2.7.0",
+    "@wordpress/element": "^2.11.0",
+    "@wordpress/i18n": "^3.9.0",
+    "@wordpress/url": "^2.11.0",
+    "colorthief": "^2.3.0",
     "draft-js": "^0.11.4",
     "draft-js-export-html": "^1.4.1",
     "draft-js-import-html": "^1.4.1",
     "draftjs-filters": "^2.3.0",
     "mousetrap": "^1.6.5",
+    "polished": "^3.4.4",
     "prop-types": "^15.7.2",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
     "react-modal": "^3.11.1",
+    "react-moveable": "^0.17.0",
     "resize-observer-polyfill": "^1.5.1",
+    "styled-components": "^5.0.1",
     "uuid": "^7.0.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.8.4",
     "@babel/core": "^7.8.4",
     "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/plugin-proposal-object-rest-spread": "^7.8.3",
-    "@babel/plugin-transform-react-jsx": "^7.8.3",
     "@prettier/plugin-xml": "^0.7.2",
```

Comment:
Which files are we using this for btw? Just curious.

Reply:
`@prettier/plugin-xml`? All XML files. As of now that includes only `phpcs.xml` and `phpunit.xml.dist`.

Reply:
Ah, right, forgot about these files.